hack/get-machine-run-tests.sh

```diff
@@ -785,6 +791,7 @@ ANSIBLE_LOG_PATH=/tmp/ansible-logging.log ansible-playbook -vvv --become \
   -e openshift_logging_es_allow_external=${ES_ALLOW_EXTERNAL:-True} \
   -e openshift_logging_es_ops_allow_external=${ES_OPS_ALLOW_EXTERNAL:-True} \
   -e oreg_url='openshift/origin-\${component}:'"\${release_commit}" \
+  -e skip_sanity_checks=true \
```

Comment:
Is there a way to fix this?  If you skip all sanity checks, you could end up with a failed installation due to lack of resources - or, worst case, install seems to pass, but openshift and logging will have strange problems that are difficult to debug.